packages/frontend-api/src/Model/Token/TokenAuthenticator.php

```diff
@@ -26,10 +29,12 @@ class TokenAuthenticator extends AbstractAuthenticator
     /**
      * @param \Shopsys\FrontendApiBundle\Model\Token\TokenFacade $tokenFacade
      * @param \Shopsys\FrontendApiBundle\Model\User\FrontendApiUserProvider $frontendApiUserProvider
+     * @param \Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserFacade $customerUserFacade
      */
     public function __construct(
         protected readonly TokenFacade $tokenFacade,
         protected readonly FrontendApiUserProvider $frontendApiUserProvider,
+        protected readonly CustomerUserFacade $customerUserFacade,
```

Comment:
this should be mentioned in the upgrade notes

Reply:
added 2b5a080058